File: Python PowerPoint Examples/SlidesApiDemo/getSlidesList.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.slides_api.SlidesApi(configuration)

fileName = "GetSlidesList.pptx"
password = ""
storage = ""
inputFolder = "input"
result = api.get_slides_list(fileName, password=password, storage=storage, folder=inputFolder)

File: PHP PowerPoint Examples/SlidesApiDemo/getSlidesList.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\SlidesApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new SlidesApi($configuration);

$name = "SlidesList.pptx"; 
$password = null; 
$folder = "Powerpoint"; 
$storage = null; 
$result = $apiInstance->getSlidesList($name, $password, $folder, $storage);